lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -33,9 +33,10 @@ function multiplyFractions(node) {
   const numeratorArgs = [];
   const denominatorArgs = [];
   node.args.forEach(operand => {
-    if (Node.Type.isOperator(operand, '/')) {
-      numeratorArgs.push(operand.args[0]);
-      denominatorArgs.push(operand.args[1]);
+    if (Node.Type.isFraction(operand)) {
+      const fraction = Node.Type.getFraction(operand);
```

Comment:
Omg good catch! * facepalm *

I modified the`isFraction`/`getFraction` functions to _not_ allow unary minus for now, since now that you've pointed this out, it seems dangerous.

It looks like a simple case like -1/2 works fine, because minus does go on the numerator. The case that doesn't work is -(1/2). I thought for a while but couldn't think of a clean way to handle this right now, so I'm punting for the moment. Added a TODO comment in the tests. Let me know if you have any ideas though.